buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/ElasticsearchNode.java

```diff
@@ -931,8 +969,17 @@ private void createConfiguration() {
                 )
                     .map(entry -> entry.getKey() + ": " + entry.getValue())
                     .collect(Collectors.joining("\n"))
-                    .getBytes(StandardCharsets.UTF_8)
+                    .getBytes(StandardCharsets.UTF_8),
+                StandardOpenOption.TRUNCATE_EXISTING, StandardOpenOption.CREATE
             );
+            Path jvmOptions = configFile.getParent().resolve("jvm.options");
```

Comment:
I'm confused as to why this is necessary? Why are we cherry picking files here?

Reply:
We could recursively copy the contents of the directory, but these are the relevant files

Reply:
I guess I don't understand why we need to explicity copy these files from the distribution dir to the working directory?

Reply:
We are setting up a new config dir, outside of the distro dir. 
These only get copied if they don't exist, so for bwc tests they will use the old versions even after upgrade.

Reply:
I'm replacing this with walking the config dir so we don't hard-code files